DEVELOPMENT.md

```diff
@@ -1,71 +1,195 @@
-# AWS Deadline Cloud for Cinema4D Development
+# Development documentation
 
-This package has two active branches:
+This documentation provides guidance on developer workflows for working with the code in this repository.
 
-- `mainline` -- For active development. This branch is not intended to be consumed by other packages. Any commit to this branch may break APIs, dependencies, and so on, and thus break any consumer without notice.
-- `release` -- The official release of the package intended for consumers. Any breaking releases will be accompanied with an increase to this package's interface version.
-## Build / Test / Release
+Table of Contents:
 
-### Build the package
+* [Development Environment Setup](#development-environment-setup)
+* [The Development Loop](#the-development-loop)
+   * [Submitter Development Workflow](#submitter-development-workflow)
+      * [Pre-requisites](#running-the-plug-in)
+      * [Making Code Changes](#making-submitter-code-changes)
+      * [Running Tests](#running-submitter-tests)
+   * [Adaptor Development Workflow](#adaptor-development-workflow)
+      * [Running the Adaptor Locally](#running-the-adaptor-locally)
+      * [Running the Adaptor on a Farm](#running-the-adaptor-on-a-farm)
+      * [Testing the Adaptor](#testing-the-adaptor)
 
-```bash
-hatch build
-```
+## Development Environment Setup
+
+To develop the Python code in this repository you will need:
+
+1. Python 3.9 or higher. We recommend [mise](https://github.com/jdx/mise) if you would like to run more than one version
+   of Python on the same system. When running unit tests against all supported Python versions, for instance.
+2. The [hatch](https://github.com/pypa/hatch) package installed (`pip install --upgrade hatch`) into your Python environment.
+3. An install of a supported version of Cinema 4D.
+4. A valid AWS Account.
+5. An AWS Deadline Cloud Farm to run jobs on. We recommend following the quickstart in the Deadline Cloud console to create a
+   Queue with the default Queue Environment, and a Service Managed Fleet.
+
+You can develop on a Linux, MacOS, or Windows workstation.
```

Comment:
Linux isn't supported for development since [C4D doesn't support Linux workstations](https://www.maxon.net/en/requirements/cinema-4d-2025-requirements), right?

Reply:
Makes sense. Looks like a copy paste bug. It should only be Windows instances for now.